### DISCUSSION TITLE
How are we marking issues in main branch that need backporting to stable branch?

### DISCUSSION THREAD

**Discussion by johnaohara** [Hyperfoil/Horreum#363] [Q&A] [answered]
@jesperpedersen re subject, do we need a label to indicate that a PR needs to be backported to the stable branch?

**Comment by jesperpedersen** [comment, marked as answer]
The labels have been updated.

`feature` is a new feature - will only go on `master`.
`enhancement` is an enhancement to an existing feature. This can be tagged with `master` (by default) and `stable` (optional). Requires a stable ABI in order to be considered to be backported.
`bug` is a bug fix - will go on `master` and `stable` by default.
